src/scripts/install.sh

```diff
@@ -130,21 +69,25 @@ Uninstall_AWS_CLI() {
 }
```

Comment:
You have to redesign this part of the script. Remember, your root file should not have any OS-specific logic. It should be oblivious to where it is running - all OS-specific code must be inside their respective files. We can pair up on this if you get stuck.